lib/core/cmd.js

```diff
@@ -5,7 +5,7 @@ const { spawn } = require('child_process');
 module.exports = {
   exec: async (command, options={}) => {
     return new Promise((resolve, reject) => {
-      const items = command.split(' ');
+      const items = command.split(' '); //TODO problem, it is splitting even posix compliant path strings such as /Application\ Support/
```

Comment:
Indeed, thanks for pointing it out, as soon as we migrate to the cmd ts package we will get rid of this issue, see https://github.com/w3f/cmd-ts/blob/master/src/cmd.ts